packages/Algernon.package/ALGInputWrapper.class/instance/handleKeystroke..st

```diff
@@ -0,0 +1,4 @@
+forwarding
+handleKeystroke: anEvent 
+
+	" no-op "
```

Comment:
I don't really understand what this is doing and future devs won't either, so maybe a comment should be added here?